atat_internal_api.yaml

```diff
@@ -2038,6 +2038,37 @@ components:
   #          type: string
   #        lastName:
   #          type: string
+    CspInvocation:
+      required:
+        - method
+        - endpoint
+        - payload
+      properties:
+        method:
+          type: string
+          enum:
+            - GET
+            - POST
+            - PUT
+            - DELETE
+        headers:
+          type: object
+          additionalProperties:
+            type: string
+        endpoint:
+          type: string
+          format: uri
+        payload:
+          type: object
+    CspResponse:
+      properties:
+        code:
+          type: integer
+        content:
+          type: object
+            additionalProperties: true
```

Comment:
`additionalProperties: true` is true by default, could probably get away with removing it.